module/solutions/helper.js

```diff
@@ -2404,6 +2404,95 @@ module.exports = class SolutionsHelper {
     });
   }
 
+   /**
+   * List Program using organization id.
+   * @method
+   * @name assets
+   * @query {String} type - Assets type (program/solutions).
+   * @returns {Object} - Details of the program under the organization.
+   */
+
+   static queryForOrganizationSolutions(bodyData){
+
```

Comment:
try to use solution list function @praveenKDass

Reply:
resolved